test/app/teams_test.rb

```diff
@@ -91,6 +91,17 @@ def test_user_must_be_manager
     end
   end
 
+  def test_logged_in_user_can_search_teams
+    [
+      [:get, '/teams'],
+      [:get, '/teams/']
+    ].each do |verb, endpoint|
```

Comment:
If we're already looping in this file, let's keep going down that route, and consider a separate PR for separating them out.